content/documentation/shortcodes.md

```diff
@@ -0,0 +1,213 @@
+{{% /center %}}
+
+## Caption
+
+Whenever you need to use an image in your post, you should consider the `caption` shortcode to provide valuable information to the user; think in the screen readers.
+
+For the caption of the image, it can be used html, markdown and other shortcodes
+
+```
+{{%/* caption src="//url-to-the-omage" title="title of image" %}}
+caption of the image
+{{% /caption */%}}
+```
+
+_example_:
+{{% caption src="http://www.elreferente.es/source/paginas/slider/sourced-ronda-6millones.jpg" title="title of image" %}}
+caption of the image
+{{% /caption %}}
+
+## Extra width content
+
+The `grid` shortcode creates an extra-width container where you can place your stuff like images, videos, tables...
+```
+{{%/* grid */%}}
+This container has extra-width
+{{%/* /grid */%}}
+```
+
+_example_:
+{{% grid %}}
+Ut non consectetur dolor pariatur pariatur ullamco. Voluptate esse id id aute nisi eu et anim labore fugiat id laboris et. Ad magna dolore consequat veniam sit nisi consequat officia sit magna dolore adipisicing. Do ullamco in occaecat et occaecat. Dolor consequat et laboris et ut tempor esse et exercitation cupidatat. Do officia cillum amet sunt. Veniam irure mollit officia fugiat.
+{{% /grid %}}
+
+An special case of the content that can be placed inside of a `grid` shortcode is the following one: [Grid of 2 elements](#grid-of-2-elements)
+
+## Grid of 2 elements
+
+The `grid-cell` shortcode must be used inside a `grid` one. It will create a "two columns layout" inside an "extra-width container"
+
+Inside each `grid-cell` you can introduce text, markdown or other shortcodes. 
+
+```
+{{%/* grid */%}}
+{{%/* grid-cell */%}}
+SOMETHING LEFT
+{{%/* /grid-cell */%}}
+{{%/* grid-cell */%}}
+SOMETHING RIGHT
+{{%/* /grid-cell */%}}
+{{%/* /grid */%}}
+```
+
+_example_:
+{{% grid %}}
+{{% grid-cell %}}
+table:
+
+| 0 | 1 | 1 | 1 | 1 | 1 |
+|:-:|:-:|:-:|:-:|:-:|:-:
+| 1 | 0 | 4 | 3 | 3 | 3 |
+| 1 | 4 | 0 | 3 | 3 | 3 |
+| 1 | 3 | 3 | 0 | 3 | 3 |
+{{% /grid-cell %}}
+{{% grid-cell %}}
+```js
+const fillTable = (table) => {
+    for (let i = 0; i < 4; i++) {
+        const row = document.createElement('tr');
+        for (let j = 0; j < 6; j++) {
+            const cell = getRandomCell();
+            row.appendChild(cell);
+        }
+        table.appendChild(row);
+    }
+}
+```
+{{% /grid-cell %}}
+{{% /grid %}}
+
+## Anchor
+
+You can use an `anchor` shortcode to create a reference anywhere, and link to it from any other place of your post (or even outside the post)
+
+```
+{{%/* anchor "anchor-name" */%}}
+```
+
+Then, to link to that anchor you only need to write a normal link like `[text of the link](#anchor-name)`
+
+_example_:
+
+[text of the link](#anchor-name)
+
+{{% anchor "anchor-name" %}}
+The `anchor` no visual representation, it's just an _"anchor"_ ;)
+
+## codepen
+
+Whenever it is needed to display something runnable like a demo, an example or a proof of concept, it should be used inside an embedded `codepen` snippet.
```

Comment:
s/Whenever it is needed to display something runnable like a demo/Whenever displaying something runnable like a demo is needed/